internal/apis/kfd/v1alpha2/eks/create/kubernetes.go

```diff
@@ -37,10 +40,17 @@ var (
 	errVpcCIDRFromOut     = errors.New("cannot read vpc_cidr_block from infrastructure's output.json")
 	errVpcCIDRNotFound    = errors.New("vpc_cidr_block not found in infra output")
 	errVpcIDNotFound      = errors.New("vpc id not found: you forgot to specify one or the infrastructure phase failed")
+	errParsingCIDR        = errors.New("error parsing cidr")
+	errResolvingDNS       = errors.New("error resolving dns")
+	errVpcIDNotProvided   = errors.New("vpc_id not provided")
+	errCIDRBlockFromVpc   = errors.New("error getting cidr block from vpc")
+	errKubeAPIUnreachable = errors.New("kubernetes API is not reachable")
```

Comment:
```suggestion
	errKubeAPIUnreachable = errors.New("Kubernetes API is not reachable")
```

Reply:
this one makes the linter mad, errors cannot start with uppercase (makes them non chainable-friendly)

Reply:
well, in this case the linter is wrong, this is a proper noun and must start with uppercase. But not a big deal